src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/ChatMessageEditor.razor

```diff
@@ -179,7 +180,16 @@
 
             } else {
                 var repliedEntryId = relatedChatEntryId.IsNone ? (long?)null : relatedChatEntryId.LocalId;
-                readEntryId = await JSRef.InvokeAsync<long>("post", Chat.Id, text, repliedEntryId);
+                if (HostInfo.AppKind == AppKind.Maui) {
```

Comment:
Why did you change this?
Currently attachments upload works on android. With this change it won't work.